app/scripts/migrations/127.ts

```diff
@@ -0,0 +1,44 @@
+import { hasProperty, isObject } from '@metamask/utils';
+import { cloneDeep } from 'lodash';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 127;
```

Comment:
If we decide to cherry pick this to an already cut release, this version may change to something like 121.1 or 122.1 to make the numbers merge nicer.